### PR TITLE
Adding note about using Cache-Control: no-cache with fixity checks

### LINKED ISSUE #218: Create non-normative text suggesting Cache-Control for persistence fixity.
When using `Want-Digest` for persistence fixity, the client wants the server to compute (or otherwise verifiably produce) a checksum.  The client ought to be advised that it `SHOULD` use `Cache-Control: no-cache` so that [no intermediate caches/proxies will return a cached response](https://tools.ietf.org/html/rfc7234#section-5.2.1.4).

### PR DESCRIPTION
**Pull request by escowles**
Fixes #218

### CODE REVIEW

**Review by zimeon — CHANGES_REQUESTED** [review]
Looks good except `[[!RFC7234]]` should not have the `!` as this is informative (not that that actually has any effect because we do ref this RFC in normative sections...)

**Review by zimeon — APPROVED** [review]

**Review by barmintor — APPROVED** [review]

**Review by dannylamb — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -446,7 +446,9 @@
           <blockquote class="informative">
             Non-normative note: In the presence of a <code>Want-Digest</code> header, the current <code>Digest</code>
             value should be calculated to enable fixity verification. Otherwise, it is expected that ETag values may be
-            cached. See <a href="#persistence-fixity"></a>.
+            cached. Clients should use the <code>Cache-Control: no-cache</code> header (see [[RFC7234]]
```
Comment (awoods) [inline index.html:449]: Would it be safer to replace `Clients should use` to `Clients are advised to use`?

Comment (zimeon) [reply index.html:449]: I'm not sure about this because it is the designers of clients that are advised

Comment (awoods) [reply index.html:449]: To be clear, are you suggesting that the current wording expresses the intent to your liking, @zimeon ?

Comment (zimeon) [reply index.html:449]: Yes, I think the current wording is adequate (and I think it would not be quite right to say "clients are advised")

**Pull request merged**